### PR TITLE
[OSPP]: 添加 buddy-codesign

### PR DESCRIPTION
**Pull request by qzylalala**
【OSPP】将 buddy-codesign 作为 submodule 提交到 buddy-mlir

### CODE REVIEW

**Comment by zhanghb97** [comment]
Please just add the in-tree `autotuner` directory for the OSPP project (do not use the submodule in case OSPP evaluation fails), and writing all the code into the `autotuner` directory is fine for now.

**Comment by qzylalala** [comment]
> Please just add the in-tree `autotuner` directory for the OSPP project (do not use the submodule in case OSPP evaluation fails), and writing all the code into the `autotuner` directory is fine for now.

Sorry about that.
So should I just clone repo "buddy-codesign" or add files manually?

File: autotuner/mlir-cpu-runner
Comment (zhanghb97) [inline autotuner/mlir-cpu-runner:1]: Please remove the binary file.

**Review by zhanghb97 — COMMENTED** [review]
- Add license head for every Python file, see [here](https://github.com/buddy-compiler/buddy-mlir/blob/main/frontend/Python/frontend.py) as an example.
- Format all the Python files according to the [code style guide](https://buddy-compiler.github.io/Pages/Documentation/CodeStyle.html).
- Using English for the comments.

**Comment by zhanghb97** [comment]
In the future, this part will be used separately in the buddy-codesign repository, so I switched the target branch, and this PR will be merged into the co-design branch instead of the main branch.

**Review by zhanghb97 — COMMENTED** [review]
- Use English as comments.
- Check license head 80-col rule.
- Use the `black` tool format all the Python files.

**Comment by qzylalala** [comment]
> * Use English as comments.
> * Check license head 80-col rule.
> * Use the `black` tool format all the Python files.

Sorry about that.
Done now!

**Pull request merged**